<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SortPage</class>
 <widget class="QWidget" name="SortPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
color:white;
border:0;
padding:10px;
padding-left:30px;
padding-right:30px;
font-size:30px;
background-color:#535c68;
}

QPushButton:hover{
background-color:#576574;
}

QComboBox{
padding:5px;
font-size:25px;
color:white;
}

QComboBox QAbstractItemView
{
padding:5px;
font-size:25px;
color:white;
}

</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="customWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="startVisualizationButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
color:lime;
}</string>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopVisualizationButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
color:red;
}</string>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="toMenuButton">
       <property name="text">
        <string>Menu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sortMethodCombo">
       <item>
        <property name="text">
         <string>Bubble sort</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Insertion sort</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Selection sort</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Merge sort</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Quick sort</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
